cypher/cypher-docs/src/docs/graphgists/query-tuning/advanced-query-tuning-example.asciidoc

```diff
@@ -290,3 +290,6 @@ Predicates that will not work:
 * Existence (eg. `WHERE exists(n.email)`) because no property type information is given
 
 An existence constraint does not include any type information either and will thus not be enough to trigger _index-backed order by_.
+
+As of Neo4j 4.1 queries with parameters can trigger _index-backed order by_.
+The only exception are queries with any parameters of type `Float` or `Point`.
```

Comment:
Same as in the 4.0 version, can it be clarified that it only applies when the property of the index is the parameter?